tests/manage/test_pvc_disruptive.py

```diff
@@ -22,38 +28,182 @@ class BaseDisruption(ManageTest):
     pvc_obj = None
     storage_type = None
 
+    def create_pvcs(self):
+        """
+        Create PVCs
+
+        Returns:
+            list: List of PVC objects
+        """
+        # Create 5 PVCs
+        pvc_objs = helpers.create_multiple_pvcs(
+            sc_name=self.sc_obj.name, namespace=self.namespace, number_of_pvc=5
+        )
+        return pvc_objs
+
+    def num_of_resources_added(self, func_to_use, previous_num):
+        """
+        Wait for new resources to be created and find the number of new
+        resources created.
+
+        Args:
+            func_to_use (function): Function to be used to fetch resource info
+            previous_num (int): Previous number of resources
+
+        Returns:
+            int: Number of newly added resources
+        """
+        for sample in TimeoutSampler(10, 1, func_to_use, self.namespace):
+            if func_to_use == get_all_pvcs:
+                current_num = len(sample['items'])
+            else:
+                current_num = len(sample)
+            if current_num > previous_num:
```

Comment:
this will return the difference between the current number of resources and the previous one BUT, it will do it for the moment current_num is bigger than previous_num. It means that in the following seconds, the difference could increase and it won't be reflected by the returned value

Reply:
This is to check whether PVC/Pod creation has actually started. It intention is not to fetch the exact number of new PVC/Pod.

Reply:
If the intention is to check if PVC/Pod creation has started, then I think it should return True or False, and the name of the function should reflect this intention, otherwise it will be misleading to people who might want to re-use it.

Reply:
@ebondare Returning the number of new pvc/pod for  logging purpose in test case. To give message that these many new resources are found in latest check.

Reply:
@jilju I agree with @ebondare - this info we'll log won't be accurate due to the reason I explained above

Reply:
@ebenahar  
It is true that the actual number at the time of logging can be different from the calculated number. The plan is to make sure that at least 1 new resource is created before proceeding to next step. It is possible to write log info without specifying any number. The number will be an additional information.
Reason for logging this information:
The total number of 5 pvc is given based on observation. The calculated new resource number was 1 most of the time or 2 maximum. If we see that number is 4 or 5 due to speedy resource creation in later versions, the total number of pvc has to be increased.

Reply:
@ebenahar If it is inappropriate to log the number for this reason, we can remove that.
Please suggest.

Reply:
@jilju I think it would be better to return bool here without logging the number of new resources.

Reply:
Done.